### DISCUSSION TITLE
.zip manifest example

### DISCUSSION THREAD

**Discussion by infinitepower18** [microsoft/winget-cli#2501] [General] [answered]
Are there any examples of .zip manifests? Trying to experiment with it but not able to figure out how to make a manifest for .zip files.

**Comment by Trenly** [comment, marked as answer]
As of yet there are no examples, since Zip isn’t in the stable release yet, only in preview. As we get closer to stable, I'd expect to see more examples get pushed up; However, the winget-pkgs won’t start accepting Zip manifests until several days after support is pushed out to release clients.

In the meantime, I'll find a zip installer and make an example on my fork of the repo

**Reply by JaxsonLizeth** [reply]
understood what you try to say

**Reply by Trenly** [reply]
https://github.com/microsoft/winget-pkgs/tree/master/manifests/w/WindowsPostInstallWizard/UniversalSilentSwitchFinder/1.5.0.0

**Reply by kdpuvvadi** [reply]
@Trenly should the hash be nested installer or zip file itself?

**Reply by kdpuvvadi** [reply]
Nevermind, it's zip file's hash.

**Reply by mdanish-kh** [reply]
> Nevermind, it's zip file's hash.

You might wanna see the discussion here https://github.com/microsoft/winget-cli/pull/2675

**Reply by soroshsabz** [reply]
ITNOA

@Trenly I think at now zip is stable and we need to have specific doc for that

thanks
